core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -724,19 +724,29 @@ export class BaseUser {
               path.basename(dirName)
             )
           : path.join(testPath, dirName, 'diff-snapshots'),
+        storeReceivedOnFailure: true,
+        customReceivedPostfix: '',
+        customReceivedDir: __dirname.startsWith('/home/runner')
+          ? path.join(
+              '/home/runner/work/oppia/oppia/core/tests/puppeteer-acceptance-tests/diff-snapshots',
```

Comment:
Consider extracting this into a named constant for easier readability/understandability.